checkers/mongo.go

```diff
@@ -0,0 +1,99 @@
+package checkers
+
+import (
+	"fmt"
+	"github.com/globalsign/mgo"
+)
+
+type MongoConfig struct {
+	Auth       *MongoAuthConfig
```

Comment:
Please add comments around each field, indicating whether it's required, optional and how the field will influence the checker.

Something similar to this: https://github.com/InVisionApp/go-health/blob/master/checkers/sql.go#L36-L82

Reply:
Description added, redis checker used as template